mlflow/server/handlers.py

```diff
@@ -28,8 +28,12 @@ def _get_store():
     global _store
     if _store is None:
         store_dir = os.environ.get(FILE_STORE_ENV_VAR, os.path.abspath("mlruns"))
-        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, store_dir)
-        _store = FileStore(store_dir, artifact_root)
+        artifact_root = os.environ.get(ARTIFACT_ROOT_ENV_VAR, None)
+        if _is_db_uri(store_dir):
+            from mlflow.store.sqlalchemy_store import SqlAlchemyStore
+            return SqlAlchemyStore(store_dir, artifact_root)
```

Comment:
reusing `FILE_STORE_ENV_VAR` and `--file-store` option for initialization for now.

Reply:
This one I'm not sure there's a reason to avoid updating this one -- I think this is only used to communicate between `mlflow/server/__init__.py` and `mlflow/server/handlers.py` -- in other words, it is not expected that any user actually sets it (we don't document it).